src/actors/upgrade/vcs_repos/vcs_detection/check

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+import sys
+import json
+
+
+def load_inputs():
+    "Load all inputs and return dictionary with all input channels."
+    inputs = {}
```

Comment:
Optional: May I suggest something simpler in my opinion? (It's just my opinion :) )
```
try:
    return json.load(sys.stdin) if not sys.stdin.isatty() else {}
except (TypeError, ValueError):
    return {}
```

Reply:
I like that, but in this case you ignore errors from `json.load` which IMO should be propagated as we expect that input should be always valid, otherwise we will have troubles to detect such type of errors. What do you think about that?

Anyway, function like that could be part of API. It is pretty small, but does exactly what usually you will want to do.